src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, Grid } from 'semantic-ui-react';
+
+const SingleArticle = ({ singleArticle, getArticles }) => {
+  return (
+    <>
+      {singleArticle && (
```

Comment:
put the singleArticle as an empty object in your initial state, then you can remove this conditional

Reply:
Yep we did that now. Thank you.